osmhelper/osm_stops.py

```diff
@@ -1,11 +1,7 @@
 # coding=utf-8
 
-import overpy
 import re
-import sys
 from math import cos, sin, atan2, sqrt, radians, degrees
-from transitfeed import util
-
 
 STOP_REGEX = re.compile('(TICAN|TISAN|TICEN|TITRI|TILAG|TIRIO|TISAC).*')
```

Comment:
Sure, let's keep this small and don't change it any more than really necessary. I am just trying to find the reason, why the trips aren't created. And so far it looks like the stops can't be matched due to the normalization failing. Maybe that is relating to the new way of storing the stop objects?